src/main/java/core/basesyntax/customexception/ValidationExceptionIncorrectValue.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.customexception;
+
+public class ValidationExceptionIncorrectValue extends RuntimeException {
```

Comment:
every custom exception should end on word "Exception"
```suggestion
public class UserValidationException extends RuntimeException {
```